Src/Generic/Extensions/UIControl+DelayedTouchCancel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UIControl+DelayedTouchCancel.swift
+//  Ugglan
+//
+//  Created by Gustaf Gunér on 2019-05-29.
+//
+
+import Flow
+import Foundation
+import UIKit
+
+extension UIControl {
+    func delayedTouchCancel(delay: Double = 0.2) -> Signal<Bool> {
```

Comment:
This could be a `Signal<Void>` right? or does the Bool mean something?

Reply:
Yep yep, you're right